src/main/java/org/wildfly/security/auth/provider/jdbc/mapper/PasswordKeyMapper.java

```diff
@@ -74,7 +78,8 @@
 import static org.wildfly.security.password.interfaces.UnixSHACryptPassword.ALGORITHM_CRYPT_SHA_512;
 
 /**
- * A {@link KeyMapper} that knows how to map columns to a {@link org.wildfly.security.password.Password} instance.
+ * <p>A {@link KeyMapper} that knows how to map columns from a SQL query to attributes of specific {@link org.wildfly.security.password.Password} type
```

Comment:
The leading ``<p>`` is not needed; they are only needed to separate paragraphs.  I've been going through and doing cleanup, so it doesn't really matter for the purposes of this review, but just letting you know for future reference.